tests/dragonfly/replication_test.py

```diff
@@ -2626,3 +2626,34 @@ async def test_replica_of_replica(df_factory):
         await c_replica2.execute_command(f"REPLICAOF localhost {replica.port}")
 
     assert await c_replica2.execute_command(f"REPLICAOF localhost {master.port}") == "OK"
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
+async def test_big_containers(df_factory):
+    master = df_factory.create(proactor_threads=4)
+    replica = df_factory.create(proactor_threads=4)
+
+    df_factory.start_all([master, replica])
+    c_master = master.client()
+    c_replica = replica.client()
+
+    logging.debug("Fill master with test data")
+    seeder = StaticSeeder(
+        key_target=20,
+        data_size=4000000,
+        collection_size=1000,
+        variance=100,
+        samples=1,
```

Comment:
samples is into how many sets we divide key_target to apply variance, maybe use > 1 to have mixed-size values

Reply:
I wanted to have only one size, big or small or average maybe we can add one more test later to cover other cases